pkg/image/airgap.go

```diff
@@ -675,6 +700,107 @@ func reportWriterWithProgress(imageInfos map[string]*imagetypes.ImageInfo, repor
 	return pipeWriter
 }
 
+type OCIArtifactFile struct {
+	Name      string
+	Path      string
+	MediaType string
+}
+
+func PushEmbeddedClusterArtifacts(airgapBundle string, opts imagetypes.PushEmbeddedArtifactsOptions) error {
```

Comment:
Nit pick: can we make this function more generic where it takes a directory and push files in it as artifacts? Can then pass the `embedded-cluster` directory as a parameter, and rename `PushEmbeddedArtifactsOptions` to be more generic.

Reply:
Something like this:
```suggestion
func PushOCIArtifacts(airgapBundle string, opts imagetypes.PushOCIArtifactsOptions) error {
```